extensions/mssql/src/main.ts

```diff
@@ -217,6 +218,15 @@ async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionPr
 	}
 }
 
+async function exists(path: string): Promise<boolean> {
+	try {
+		await fs.access(path);
+		return true;
+	} catch (e) {
+		return false;
+	}
+}
```

Comment:
Generally prefer to have these stuck in a utils file so it's more discoverable/reusable

Reply:
The duplication is over different extensions. It's generally bad practice to have typescript files shared across multiple projects by just file paths.

Reply:
But within the same project I'll verify that it isn't used multiple times.

Reply:
Yeah that was definitely my intention. Thanks!